tests/component-tests/textfieldSlider/js/TextfieldSliderTests.js

```diff
@@ -27,45 +27,48 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 
         jqUnit.test("Test Init", function () {
             jqUnit.expect(8);
-            var textfieldSlider = fluid.tests.textfieldSlider.createTextfieldSlider({model: {value: 15}});
-            jqUnit.assertEquals("Slider value is set to input value", 15, $(".flc-textfieldSlider-slider").slider("value"));
+            var that = fluid.tests.textfieldSlider.createTextfieldSlider({model: {value: 15}});
```

Comment:
After we have factored out the two varieties of slider, this can become two tests